test/irmin-pack/migration.ml

```diff
@@ -19,9 +19,7 @@ let info () = Irmin.Info.empty
 
 let root_V1_source, root_V1, root_V2 =
   let ( / ) = Filename.concat in
-  ( "test" / "irmin-pack" / "data" / "version_1",
-    "_build" / "version_1",
-    "_build" / "version_2" )
+  ("data" / "version_1", "_build" / "version_1", "_build" / "version_2")
```

Comment:
I changed this as `dune runtest` runs in `_build/default/test/irmin-pack` where the `data/version_1` is copied.  

`dune exec` runs in `irmin`  and moreover the deps stanza only copies the dependency when we run `runtest`, but not for `dune exec`. So I think we cannot have both `dune runtest` and `dune exec -- ./test/irmin-pack/test.exe test migration 0` working, I opted for the first.